### ISSUE TITLE
User Review

### ISSUE THREAD

**Issue by edavidj** [OpenLiberty/draft-guide-microprofile-reactive-messaging-rest#14]
- [x] Record the time it takes to go through the entire guide
- [x] Test each step in the guide is clear and easy to understand
- [x] Ensure no logic errors
- [x] Resolve anything that does not work
- [x] Resolve anything usability issues (confusing or hard to follow)
- [x] Achieve basic understanding/working knowledge of the topic in the guide

Comments:
- Time: 25 min
- As someone unfamiliar with these concepts beforehand the note: `NOTE: If you want the message to be sent to all the System microservices then configure the unique group.id for all of the instances` didn't really mean much to me. Might be worth making it a little more explicit what applying that change looks like/does?
- When running the command `mvn package` the build failed for me. Once I updated line 95 in `InventoryServiceIT.java` from the `getSystemProperty` method to `updateSystemProperty` the build passed.
- Everything else worked fine and was easy to understand, I like the addition of scripts for some of the tedious steps 👍

**Comment by MaiHameed** [comment]
> As someone unfamiliar with these concepts beforehand the note: NOTE: If you want the message to be sent to all the System microservices then configure the unique group.id for all of the instances didn't really mean much to me. Might be worth making it a little more explicit what applying that change looks like/does?

It's too complicated to explain in the guide, so it was written as is intentionally. We're leaving it as is.

**Comment by MaiHameed** [comment]
Addressed with #16, closing
